### ISSUE TITLE
Implement uninstall script or uninstall argument

### ISSUE THREAD

**Issue by RedBearAK** [canadaduane/meta-mac#1]
Please implement a proper uninstall procedure either as a separate script or just an argument that can be passed to the install script to initiate a full uninstall/disabling of Meta Mac.

**Comment by canadaduane** [comment]
Yes, good idea. When I started this project at the beginning of the week, the external Ubuntu repository that deafmute maintains for `interception` did not have `impish` support yet. Deafmute was kind enough to update that for us though, so it would make sense to make Meta Mac follow better install/uninstall procedure.
